src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
+  padding: 32px 16px 16px;
+  height: 406px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The `transform: scale(120%);` may cause the card to overflow its container or overlap with other elements when scaled. Consider reviewing the design to ensure that the hover effect does not disrupt the layout. For example, you could use a smaller scale value or adjust the layout to accommodate the scaling.